### ISSUE TITLE
Keyboard navigation with arrows is limited

### ISSUE THREAD

**Issue by MartinZikmund** [unoplatform/uno#9392]
### Current behavior

On Skia, it is not possible to navigate a whole ListView via arrows - it stops working shortly after leaving the screen bounds.

### How to reproduce it (as minimally and precisely as possible)

See Uno.IconFontTester

**Comment by ramezgerges** [comment]
This seems to be already working. I tested with `<ListView ItemsSource="qwertyuiopasdfghjklzxcvbnm1234567890" />`.

**Comment by ramezgerges** [comment]
Nope, this reproduced again for me today. It seems to need a big list (100+ items), but I still can't reliably produce it

**Comment by MartinZikmund** [comment]
still relevant

**Comment by MartinZikmund** [comment]
# Repro generator

This issue was classified as not reproducible with a code sample or test.

**Reason:** Keyboard arrow navigation bug — requires complex focus/keyboard testing.
